eventhub/Module.md

```diff
@@ -246,100 +224,46 @@ Note:
 You can specify the event hub path as a parameter of the deleteEventHub method.
 This operation will return a ballerina error if the operation failed.
 
-# Samples:
+## Snippets
+Snippets of some operations.
```

Comment:
Shall we say  "Quick reference on how to use the most frequently used operations"

Reply:
Fixed in df7c75b . Thank you.